### ISSUE TITLE
remove filter to prevent On-Hold emails (465)

### ISSUE THREAD

**Issue by AlexP11223** [woocommerce/woocommerce-paypal-payments#411]
The filter skipping emails for On Hold status should be no longer required after the changes in #341

Before that change, an order was created in the status “On-Hold” and then set to “Processing” right after, causing two emails to be sent at the same time.

Since the filter is active by default, no emails will be sent anymore when the order status is “On-Hold”. This also includes scenarios where the capture mode is set to “Authorize”.

In that case, when a user performs a purchase, they will not receive an email confirming the order until the payment was captured.

### To Reproduce

1. set payment mode to “Authorize”
2. create order
3. no email will be sent to user, only after capturing the order when it moves to “Processing”
4. remove the filter mentioned below
5. create order
6. mail will be sent to the user

### Expected Behavior

When the order is set to On-Hold, the user should receive an email.
